### DISCUSSION TITLE
DAX Studio run data to Excel with no name to sheet

### DISCUSSION THREAD

**Discussion by Billbuild** [DaxStudio/DaxStudio#679] [Q&A]
After EVALUATE in the DAX Studio, the data was run into the Excel, but the sheet with the default name "query".
My question is how to give an individual name to the Excel sheet when run data to Static Excel.

Thanks a lot!

**Comment by dgosbell** [comment]
Unfortunately there is no way to do this at present. And there is no simple place to prompt for a name. I'm working on something which may enable this in the future, but it's a bit complex so I don't know when this might be available.

**Reply by Billbuild** [reply]
Thanks with your response with my question!
